examples/nft-fee-transaction/project/Dependencies.scala

```diff
@@ -0,0 +1,44 @@
+import sbt.*
+
+object Dependencies {
+
+  object V {
+    val tessellation = "99.99.99" //TODO: Update with the official version with FeeTransactions
```

Comment:
do you want to update this to a tagged version?

Reply:
We don't have a fixed version in testnet, this should be updated once we migrate to intnet

Reply:
I think it would be `3.0.0`. but if we can, let's wait for release